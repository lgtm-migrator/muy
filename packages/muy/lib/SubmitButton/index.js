import React from "react"
import Button from "@material-ui/core/Button"

const SubmitButton = ({ id, name, ...props }) => (
  <Button id={id} name={name} type={"submit"} {...props} />
)

SubmitButton.defaultProps = {
  id: "submit",
  name: "submit",
}

export default SubmitButton
