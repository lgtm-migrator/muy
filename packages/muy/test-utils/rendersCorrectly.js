import renderer from "react-test-renderer"

const rendersCorrectly = component => {
  const tree = renderer.create(component).toJSON()
  return expect(tree).toMatchSnapshot()
}

export default rendersCorrectly
