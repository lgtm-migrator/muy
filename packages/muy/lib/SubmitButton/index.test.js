import React from "react"
import rendersCorrectly from "../../test-utils/rendersCorrectly"
import SubmitButton from "."

describe("<SubmitButton /> component", () => {
  it("should render correctly with no props", () => {
    rendersCorrectly(<SubmitButton />)
  })
})
