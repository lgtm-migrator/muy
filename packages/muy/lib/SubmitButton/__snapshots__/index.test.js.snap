// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubmitButton /> component should render correctly with no props 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-text"
  disabled={false}
  id="submit"
  name="submit"
  onBlur={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="submit"
>
  <span
    className="MuiButton-label"
  />
</button>
`;
